public/views/address.html

```diff
@@ -1,9 +1,9 @@
 <section data-ng-controller="AddressController">
+  <link rel=StyleSheet href="../src/css/common.css" type="text/css" media="screen">
   <div class="secondary_navbar hidden-xs hidden-sm" scroll data-ng-class="{'hidden': !secondaryNavbar}" data-ng-if="address.balance >= 0" data-ng-init="hideSNavbar=0">
     <div class="container" data-ng-if="!hideSNavbar">
       <div class="col-md-8 text-left">
-        <h3>Address</h3> {{address.address}}
-        <span class="btn-copy" clip-copy="address.address"></span>
+        <h2>Address:&nbsp;{{address.address}}</h2>
```

Comment:
Now we have:
![before](https://cloud.githubusercontent.com/assets/22824146/21464202/7b64bcfe-c976-11e6-8eb4-1836ab3db632.png)
After your changes:
![after](https://cloud.githubusercontent.com/assets/22824146/21464203/7e635b4a-c976-11e6-9596-5e793fc32ede.png)

I think that first one looks better, but maybe it's just a personal preference.